dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -137,13 +141,9 @@ boolean isSystemClass(String className) {
     }
   }
 
-  /**
-   * Returns class file names defined in the jar file.
-   *
-   * @param jarPath absolute path to the jar file
-   */
-  ImmutableSet<String> classesDefinedInJar(Path jarPath) {
-    return jarFileToClassFileNames.get(jarPath);
+  /** Returns class file names defined in the class path entry. */
```

Comment:
"defined" is perhaps not the right word here. Perhaps "Found"? Or maybe just `classNamesInJar`